contracts/erc20/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+    "overrides": [],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "./tsconfig.json",
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": ["@typescript-eslint"],
+    "ignorePatterns": [
+        "lib/**/*",
```

Comment:
no lib., migrations, and generated-wrappers in root of this package, keep only the test/*